<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDOValueSliderEditDlg</class>
 <widget class="QDialog" name="SDOValueSliderEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Слайдер</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbData">
     <property name="title">
      <string>Переменная</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="lblType">
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblIndex">
        <property name="text">
         <string>Индекс</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbIndex">
        <property name="prefix">
         <string>0x</string>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8192</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbSubIndex">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblNodeId">
        <property name="text">
         <string>Устройство</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSubIndex">
        <property name="text">
         <string>Под-индекс</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbNodeId">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gbPen">
     <property name="title">
      <string>Внешний вид</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="2">
       <widget class="QToolButton" name="tbHandleColorSel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblGrooveColor">
        <property name="text">
         <string>Цвет канавки</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="tbTroughColorSel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="tbGrooveColorSel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QFrame" name="frTextColor">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="sbPenWidth">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QToolButton" name="tbTextColorSel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblScaleColor">
        <property name="text">
         <string>Цвет шкалы</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblLineWidth">
        <property name="text">
         <string>Толщина линии</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QFrame" name="frHandleColor">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="cbTrough">
        <property name="text">
         <string>Впадина</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QFrame" name="frScaleColor">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QSpinBox" name="sbSteps">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lblSteps">
        <property name="text">
         <string>Число шагов</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblHandleColor">
        <property name="text">
         <string>Цвет рукоятки</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QToolButton" name="tbScaleColorSel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QFrame" name="frGrooveColor">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="cbGroove">
        <property name="text">
         <string>Канавка</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="frTroughColor">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblTextColor">
        <property name="text">
         <string>Цвет текста</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTroughColor">
        <property name="text">
         <string>Цвет впадины</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblOrientation">
        <property name="text">
         <string>Положение</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbOrientation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbRange">
     <property name="title">
      <string>Диапазон</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lblRangeMin">
        <property name="text">
         <string>Минимум</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sbRangeMin">
        <property name="minimum">
         <double>-4294967296.000000000000000</double>
        </property>
        <property name="maximum">
         <double>4294967296.000000000000000</double>
        </property>
        <property name="value">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblRangeMax">
        <property name="text">
         <string>Максимум</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="sbRangeMax">
        <property name="minimum">
         <double>-4294967296.000000000000000</double>
        </property>
        <property name="maximum">
         <double>4294967296.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="gbPos">
     <property name="title">
      <string>Расположение</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lblPosRow">
        <property name="text">
         <string>Строка</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbPosRow">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblSizeRow">
        <property name="text">
         <string>Строк</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="sbSizeRow">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPosCol">
        <property name="text">
         <string>Столбец</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbPosCol">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblSizeCol">
        <property name="text">
         <string>Столбцов</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="sbSizeCol">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SDOValueSliderEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SDOValueSliderEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
